src/TableFunctions/TableFunctionAzureBlobStorage.h

```diff
@@ -46,7 +46,9 @@ class TableFunctionAzureBlobStorage : public ITableFunction
         return {"_path", "_file"};
     }
 
-    static StorageAzureBlob::Configuration parseArgumentsImpl(ASTs & args, const ContextPtr & context, bool get_format_from_file = true);
+    virtual void parseArgumentsImpl(ASTs & args, const ContextPtr & context);
```

Comment:
Comment in line 16 has incorrect signature (it was copied from s3), please, correct it.